code/style.css

```diff
@@ -1,7 +1,322 @@
+  border-radius: 0.5em;
+  font-family: 'Work Sans', sans-serif;
+  opacity: 80%;
+}
+.form:hover{
+  opacity: 100%;
+}
+
+input{
+  border: solid;
+  border-width: 1px;
+  font-size: 1em;
+  padding: 1em;
+  margin-bottom: 1em;   
+}
+
+.textblock{
+  width: 90%;
+}
+
+input:valid{
+  box-shadow:0.25em 0.25em 0.25em rgb(76, 205, 76);
+}
+
+input:invalid{
+  box-shadow: 0.2em 0.2em 0.2em rgb(232, 89, 89);
+}
+
+.label {
+  display: block;
+  padding: 0.5em;
+  text-align: left;
+}
+.name{
+  border-style: solid;
+  text-align: left;
+}
+
+.button{
+  margin: 1em;
+  padding: 1em;
+  border-radius: 1em;
+  background-color: #F5D081;
+}
+
+.accordion{
+  border-style: solid;
+  padding: 2em;
+  width: 70%;
+  font-family: 'Work Sans', sans-serif;
+  font-size: 25px;
+  background-color: #B8CABF;
+  border-radius: 0.5em;
+  opacity: 80%;
+}
+
+.accordion:hover{
+  opacity: 100%;
+}
 
 .active + .answer {
-  /* some styling here */
+  max-height: 5em;
+  font-style: italic;
+  font-size: 18px;
+}
+  
+.active {
+   font-weight: bold;
+}
+  
+.answer{
+   overflow: hidden;
+   max-height: 0;
+   transition: 1s ease-in-out;
+  }  
+
+.question:after {
+    /* Unicode character for "plus" sign (+) */
+    content: '\002B';
+    color: #19294C;
+    float: right;
+  }
+
+.active:after {
+  /* Unicode character for "minus" sign (-) */
+  content: "\2212";
+  color: #19294C;
+}
+
+/* This is the third section where a row of images appear */
+.img-container {
+    display: grid;
+    flex-wrap: wrap;
+    grid-template-columns: repeat(3, 1fr);
+  }
+    
+.grid-item-img {
+    grid-column: span 1;
+    display: grid;
+    flex-direction: column;
+  }
+  
+footer{
+    display: flex;
+    font-family: 'Work Sans', sans-serif;
+    justify-content:space-between;
+    flex-wrap: wrap;
+    align-items: center;
+    background-color: #6A8187 ;
+    padding: 5em;
+    color: white;
+    font-size: 20px;
+  }
+  
+@media screen and (max-device-width: 640px) {
+
+header{
+  padding-bottom: 2em;
+}
+
+.hero-text-box {
+  width: 90%;
+  margin-left: 2em;
+}
+
+.hero-title{
+  margin-bottom: 9em;
+  font-size: 32px;
+}
+
+.h2{
+  margin-left: 0.5em;
+  font-size: 20px;
+}
+
+.header-text{
+  font-size: 15px;
+}
+
+.one{
+  display: none;
+  }
+
+.two{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    flex-wrap: wrap; 
+    gap: 1em;
+    padding: 0.1em 3em 5em 3em;
+    background-color: transparent;
+    opacity: 95%;
+  }
+
+.form{
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  border-style: none;
+  padding: 2em;
+  margin: 2em;
+  width: 100%;
+  text-align: center;
+  opacity: 100%;
+}
+
+.textblock{
+  width: 100%;
+}
+  
+input {
+  border: solid;
+  border-width: 1px;
+  font-size: 1em;
+  padding: 1em;
+  margin-bottom: 1em;   
+}
+  
+input:valid{
+      background-color: rgb(76, 205, 76);
+  }
+  
+input:invalid{
+      background-color: rgb(232, 89, 89);
```

Comment:
The contrast between the text and background color is not so good. I can barely read the text after the background turns red. It's good though that you have some styling to indicate invalidity.